doc/md/versioned/diff.mdx

```diff
@@ -98,11 +98,11 @@ table "users" {
 
 Then, run `atlas migrate diff`:
 
-```bash
+```shell
```

Comment:
Are you sure they have the same formatting? 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/7413593/188111646-cf96cd3c-4775-488b-a303-6c052691ac5e.png">

Reply:
I haven't checked, but we use `shell` everywhere else, at least it's what I always did.

Reply:
I remember that I checked that and bash looked good. Add a screenshot, please.

Reply:
![image](https://user-images.githubusercontent.com/12862103/188143073-cfeae19a-dce5-4efc-8006-251a4de5c078.png)

Reply:
looks exactly the same